public/js/gulliver.es6.js

```diff
@@ -278,13 +278,38 @@ function setupConfig() {
   }
 }
 
+function setupMessaging() {
+  const messaging = new Messaging();
+  messaging.init();
+
+  const checkbox = document.getElementById('notifications');
```

Comment:
Created a NotificationCheckbox class on a separate file. I think we should start discussing using web-components in this scenario: we could encapsulate the Html, CSS and JS in a single file.